packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -1724,6 +1724,7 @@ interface SqlServerInstanceEngineProps {
 interface SqlServerInstanceEngineBaseProps {
   readonly engineType: string;
   readonly version?: SqlServerEngineVersion;
+  readonly engineFamily?: string
```

Comment:
From the CloudFormation documentation it looks like this could be an enum or an enum like class. Let's use one of those to enforce values here.

Reply:
Actually, is there any instance where this wouldn't just be `SQLSERVER`? If not, we might as well just set that directly in `SqlServerInstanceEngineBase`.